src/object/srv_obj_migrate.c

```diff
@@ -2212,6 +2212,11 @@ migrate_enum_unpack_cb(struct dss_enum_unpack_io *io, void *data)
 	int			rc = 0;
 	int			i;
 
+	if (!daos_oclass_is_valid(daos_obj_id2class(io->ui_oid.id_pub))) {
+		D_WARN("Skip invalid "DF_UOID".\n", DP_UOID(io->ui_oid));
```

Comment:
is there a reasonable possibility to get invalid oid?

Reply:
Not sure yet, but I am adding some assertion and debug message in this patch.